Client/Frontend/Sync/BraveCore/History/HistoryManager.swift

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Data
+import BraveCore
+import BraveShared
+import CoreData
+import Shared
+
+class HistoryManager: NSObject {
```

Comment:
Same as above, no need for this to be an `NSObject` subclass

Reply:
Removed at https://github.com/brave/brave-ios/pull/4024/commits/34b14995e0c684215a0690d4a1adc9bc0632cfac